r_markdown/emissions.Rmd

```diff
@@ -683,12 +699,19 @@ ___
 ## Combine pollutants
 
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
-  emissions <- dplyr::select(co_co2_ch4, id, pol, mass_emitted, mass_carbon) %>%
-               dplyr::bind_rows(dplyr::select(carbonyls, id, pol, mass_emitted, mass_carbon)) %>%
-               dplyr::bind_rows(dplyr::select(ecoc, id, pol, mass_emitted, mass_carbon)) %>%
-               dplyr::bind_rows(dplyr::select(grav, id, pol, mass_emitted, mass_carbon)) %>%
-               dplyr::bind_rows(dplyr::select(ions, id, pol, mass_emitted, mass_carbon)) %>%
-               dplyr::bind_rows(dplyr::select(voc, id, pol, mass_emitted, mass_carbon))
+  emissions <- dplyr::select(co_co2_ch4, id, pol, mass_emitted, mass_carbon, qc) %>%
+               dplyr::bind_rows(dplyr::select(carbonyls, id, pol, mass_emitted, mass_carbon, qc)) %>%
+               dplyr::bind_rows(dplyr::select(ecoc, id, pol, mass_emitted, mass_carbon, qc)) %>%
+               dplyr::bind_rows(dplyr::select(grav, id, pol, mass_emitted, mass_carbon, qc)) %>%
+               dplyr::bind_rows(dplyr::select(ions, id, pol, mass_emitted, mass_carbon, qc)) %>%
+               dplyr::bind_rows(dplyr::select(voc, id, pol, mass_emitted, mass_carbon, qc))
+
+  emissions_g <- dplyr::select(co_co2_ch4, id, pol, mass_emitted, mass_carbon, qc) %>%
```

Comment:
I think you can make this data frame from emissions without creating the _g dataframe?